src/main/resources/api/paths/calendar/calendars.yaml

```
@@ -0,0 +1,131 @@
+get:
+  summary: Search all calendars
+  description: Get all calendars that match the given query
+  operationId: searchCalendars
+  parameters:
+    - in: query
+      name: id
+      required: false
+      style: form
+      explode: true
+      schema:
+        type: array
+        items:
+          type: string
+          format: uuid
+      description:
+        The list of calendar IDs to retrieve, sent as separate parameters (?id=aaaa&id=bbbb...).  If
+        this list is passed, calendars must have an ID in this list in addition to any additional
+        criteria.
+    - in: query
+      name: servicePointId
```

Personally, I might be tempted to pluralise these parameters to reveal the intention that these are intended to be 1 or many parameters.

(I'm not super confident about this feedback, because whilst I like the intention revealing nature of the name, I don't know if there already conventions for not doing this)

```suggestion
      name: servicePointIds
```

---

I think singular is best here, especially given that they'd be sent in the format `?servicePointId=aaaaa&servicePointId=bbbbbb`

---

Ok. (I wasn't sure what form they would be sent in, I thought the documentation said they were comma separated).

---

Spring will accept either and parse it appropriately, they're interchangable